docs/pages/serving_an_origin.mdx

```diff
@@ -55,123 +57,309 @@ Once you go through the process, locate your credential files and set the follow
+  # POSIX is the default storage type for Pelican origins
+  # and can be omitted
+  StorageType: "posix"
+
+  Exports:
+    - StoragePrefix: "/path/to/data"
+      FederationPrefix: "/your/federation/prefix"
+      # Explicitly state what capabilities you want this prefix to have
+      Capabilities: ["Reads", "Writes"]
+
+```
+and then simply run
+```bash
+pelican origin serve
+```
+Pelican will read the config file and apply it to your origin.
+
+
+Finally, origins can be configured to a limited extent with environment variables. In Pelican's environment variable model, configuration options are taken from `pelican.yaml`, flattened, and prepended with either `PELICAN_` or `OSDF_`, depending on the name of the binary you're using (ie whether you run `osdf serve` or `pelican serve` commands).
 
-This will start Pelican origin as a daemon process.
+For example, you might configure the origin's storage type by setting the environment variable `PELICAN_ORIGIN_STORAGETYPE=posix`.
+
+> Environment variable configuration does not support the same complex structures that can be built with yaml configuration, such as lists.
 
 The first time the origin is started, you will see something that looks like the following:
 
 ```console
-$ pelican origin serve -f osg-htc.org -v $PWD:/demo
+$ pelican origin serve -f https://osg-htc.org -v $PWD:/demo
 
 Pelican admin interface is not initialized
 To initialize, login at https://localhost:8444/view/initialization/code/ with the following code:
 551220
 ```
+See the [browser configuration](#login-to-admin-website) documentation section for more information about initializing your origin's web interface.
+
+### Origin and Namespace Capabilities
+
+Capabilities are the configuration options you can assign to origins and namespace prefixes to determine what kinds of access controls you want them to respect. In the previous yaml configuration, we configured the origin capabilities using the `Capabilities` list of the `Origin.Exports` block by specifying that the federation prefix `/your/federation/prefix` supports "Reads" and "Writes". This list of capabilities can be used for further control of what types of operations the namespace is willing to support. Available capabilities include:
+
+- "Reads": When included, objects from the namespace can be read with a valid authorization token.
+- "PublicReads": When set, objects from the namespace become public and require no authorization to read.
+- "Writes": When included, objects can be written back to the storage backend by Pelican. Write operations _always_ require a valid authorization token.
+- "DirectReads": When included, a namespace indicates that data being pulled from it should only be pulled through a cache and not directly by clients. This may be useful in cases where the origin isn't very performant or has to pay egress costs when data moves through it. Note that this is respected by federation central services, but may not be respected by all clients.
```

Comment:
```suggestion
- "DirectReads": When included, a namespace indicates that data being pulled from it should only be pulled through a cache and not directly by clients. This may be useful in cases where the origin isn't very performant or has to pay egress costs when data moves through it. Note that this is respected by federation central services, but may not be respected by all clients.
```

I thought this is the other way around. If `DirectReads` is not included (by default), shouldn't the director require a cache to be present or fail to redirect?

Reply:
Hah, good catch.